subsys/bluetooth/host/shell/bt.c

```diff
@@ -857,8 +856,8 @@ static void identity_resolved(struct bt_conn *conn, const bt_addr_le_t *rpa,
 	bt_addr_le_to_str(identity, addr_identity, sizeof(addr_identity));
 	bt_addr_le_to_str(rpa, addr_rpa, sizeof(addr_rpa));
 
-	shell_print(ctx_shell, "Identity resolved %s -> %s", addr_rpa,
-	      addr_identity);
+	bt_shell_print("Identity resolved %s -> %s", addr_rpa,
+		       addr_identity);
```

Comment:
Hmm, the way that it's formatted looks like clang-format, but where the line length is set to 80 instead of 100. Chances are that the author is using a misconfigured auto-formatter

Reply:
As far as I can recall, I used the in-tree `.clang-format` with some manual adjustments to align it with the previous one.
For the next push, I'll change it according to the clang-format suggestion.